packages/lablqml/lablqml.0.6/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+maintainer:   "kakadu.hafanana@gmail.com"
+authors:      "kakadu.hafanana@gmail.com"
+homepage:     "http://kakadu.github.io/lablqt/"
+bug-reports:  "https://github.com/kakadu/lablqml/issues"
+dev-repo:     "https://github.com/Kakadu/lablqml.git"
+
+tags: [ "gui" "ui" "qt" ]
+
+build: [
+  ["./configure"]
+  [make]
+]
+install: [make "install"]
+remove: [make "uninstall"]
+flags: [ light-uninstall ]
+depends: [
+  "ocamlfind"
```

Comment:
Do you use the Findlib library ? I couldn't see any reference of this in the source code. If so could you flag this dependency with the `{build}` attribute?